debugger/gdb-sgx-plugin/mman.py

```diff
@@ -0,0 +1,506 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import gdb
+import math
+import tempfile
+
+def round_up_page_size(x):
+    return ((x + 4095) // 4096) * 4096
+
+class myst_list_t:
```

Comment:
We need to put compile time asserts locking down the layout and size of these structs used by the debugger.